### PR TITLE
Add key prop to list item

### PR DESCRIPTION
**Pull request by saurabhp75**
Add key prop to list item as per React's best practices.

### CODE REVIEW

File: beta/src/content/learn/keeping-components-pure.md [hunk 1]
```diff
@@ -688,7 +688,7 @@ export default function StoryTray({ stories }) {
           {story.label}
         </li>
       ))}
-      <li>Create Story</li>
+      <li key={'create'}>Create Story</li>
```
Comment (harish-sethuraman) [inline beta/src/content/learn/keeping-components-pure.md:691]: We will need key only when we use map or other loop to render elements dynamically. I see that this `li` is outside the map so we won't need a key here.

Comment (gaearon) [reply beta/src/content/learn/keeping-components-pure.md:691]: that’s correct

Comment (saurabhp75) [reply beta/src/content/learn/keeping-components-pure.md:691]: I will close the PR then, thanx.

**Comment by saurabhp75** [comment]
Close the PR as key prop is not required for manually added individual list item.

**Pull request closed without merging**